pkg/ansible/loc.go

```diff
@@ -0,0 +1,30 @@
+package ansible
+
+import (
+	"strings"
+
+	"github.com/MaibornWolff/iac-count/pkg/metrics"
+)
+
+type LocCalculator struct {
+}
+
+func (calculator LocCalculator) Analyze(path, content string) metrics.Metric {
+	if content == "" {
+		return nil
+	}
+
+	n := 0
+	for _, r := range content {
+		if r == '\n' {
+			n++
+		}
+	}
+	if len(content) > 0 && !strings.HasSuffix(content, "\n") {
+		n++
+	}
+
+	return metrics.Loc{
+		Val: n,
+	}
```

Comment:
This is quite convoluted. How about this:

```suggestion
	return metrics.Loc{
		Val: strings.Count(content, "\n"),
	}
```